### ISSUE TITLE
'E_cdata' is not actually calculated anywhere

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#1886]
The electricity demand to cool data centers is included in the CEA results, but it is never actually calculated in the code. See below:
```
        # CALCULATE DATA CENTER LOADS
        if datacenter_loads.has_data_load(bpr):
            tsd = datacenter_loads.calc_Edata(bpr, tsd, schedules)  # end-use electricity
            tsd = datacenter_loads.calc_Qcdata_sys(tsd)  # system need for cooling
            tsd = datacenter_loads.calc_Qcdataf(locator, bpr, tsd)  # final need for cooling
        else:
            tsd['DC_cdata'] = tsd['Qcdata_sys'] = tsd['Qcdata'] = np.zeros(8760)
            tsd['mcpcdata_sys'] = tsd['Tcdata_sys_re'] = tsd['Tcdata_sys_sup'] = np.zeros(8760)
            tsd['Edata'] = tsd['E_cdata'] = np.zeros(8760)
```
If there is no data center in the building, then the demand is set to 0, but none of the scripts in the first part of the "if" statement actually calculates `tsd['E_cdata']`. So if there is a data center, this column in the csv file will be left blank.

**Comment by martin-mosteiro** [comment]
This is caused by a typo in `calc_Qcdataf`, where this variable is called `'E_data'` instead of `'E_cdata'`

**Comment by martin-mosteiro** [comment]
Solved by PR #1887
